src/3rd_party-static/jsoncpp/src/lib_json/json_reader.cpp

```diff
@@ -576,7 +576,7 @@ Reader::decodeNumber( Token &token )
                                                    : Value::maxLargestUInt;
    Value::LargestUInt threshold = maxIntegerValue / 10;
    Value::UInt lastDigitThreshold = Value::UInt( maxIntegerValue % 10 );
-   assert( lastDigitThreshold >=0  &&  lastDigitThreshold <= 9 );
```

Comment:
@LuxoftAKutsan, yes, because `lastDigitThreshold` just can't be lower then zero, because it's type is `Value::UInt`